### PR TITLE
[closes #10] Moves black and flake8 from pre-commit to actions in ci …

### LINKED ISSUE #10: What check is pre-commit and what check is for CI pipeline?
I believe one item worth discussing and agreeing at this stage is what goes into pre-commit and what gets automatically run in CI pipelines

To be honest, since personally I commit lots of WIPs, I would try to keep pre-commit as light as possible, deferring to CI pipelines all checks that may be heavy or annoying during WIPs. Beside, I believe that the "compliant" checks (mypy, linting) should be included ANYWAY in CI pipelines. Pre-commit is something that one could also disable and technically I suspect that nothing prevents not-compliant code to be pushed to the remote repository and merged. We need a central, standard and centralize check procedure to assess whether things are fine... 

Having said this, my personal opinion is that tests, mypy (given that WIPs most likely won't pass static checks) and linting would be best suited to a CI pipeline, rather than pre-commits.

### PR DESCRIPTION
**Pull request by MattiaPedro**
…pipeline

Also, I've removed the 

`on: [push, ...]`

from continous-integration.yaml @nromeoarena is that ok?

Have a look to the configurations of black and flake8 actions, if they are suitable for you.

### CODE REVIEW

File: confs/.pre-commit-config.yaml [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-        language_version: python3
-  - repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.4
-    hooks:
-      - id: flake8
-        name: Flake
-        language_version: python3
   - repo: https://github.com/jazzband/pip-tools
     rev: 6.1.0
     hooks:
```
Comment (nromeoarena) [inline confs/.pre-commit-config.yaml:4]: as outcome from issue #19 this file will be removed in the pending PR #24

**Review by nromeoarena — APPROVED** [review]
ok

File: confs/.pre-commit-config.yaml (hunk 1 above)
Comment (MattiaPedro) [reply confs/.pre-commit-config.yaml:4]: Sure, atm I've just removed what had to go in ci pipeline

**Review by NicolaDonelli — APPROVED** [review]
Fine by me.

**Review by deusebio — APPROVED** [review]

**Pull request merged**